vscode-extension/grammars/graphql.rescript.json

```diff
@@ -0,0 +1,25 @@
+{
+  "fileTypes": ["res", "resi"],
```

Comment:
I don't think regular Relay can parse these files.

Reply:
@zth what are the valid file types for rescript? i'm not familiar.

Reply:
@tbezman it's the ones you've got there, but like @captbaritone says regular Relay can't parse that, and rescript-relay has its own dedicated extension that takes care of this + integrating the lsp etc: https://github.com/zth/vscode-rescript-relay, so there's no need to do anything for ReScript specifically here.

Thanks for thinking of me though! 😃